packages/fxa-auth-server/lib/senders/oauth_client_info.js

```diff
@@ -22,18 +22,20 @@ module.exports = (log, config) => {
   /**
    * Fetches OAuth client info from the OAuth server.
    * Stores the data into server memory.
-   * @param clientId
+   * @param service
    * @returns {Promise<any>}
    */
-  async function fetch(clientId) {
+  async function fetch(service) {
     log.trace('fetch.start');
 
-    if (!clientId || clientId === 'sync') {
-      log.trace('fetch.sync');
+    // if there is no service, the client is 'sync' or (desktop) relay
```

Comment:
👍 Though (nitpick) I think this sentence could be tweaked to be a little more clear😅